backend/src/routes/v0/thread.py

```
@@ -357,3 +370,114 @@ async def delete_thread(
+    Use this endpoint to check for pending approval requests when reconnecting
+    to a thread after a connection drop, or to poll for interrupt status.
+
+    Args:
+        thread_id: The thread ID to get interrupts for
+
+    Returns:
+        InterruptList containing all pending interrupts for the thread
+    """
+    try:
+        async with get_checkpoint_db() as checkpointer:
+            service_context = ServiceContext(
```

Ruff (0.14.11)

Local variable `service_context` is assigned to but never used

Remove assignment to unused variable `service_context`

(F841)